docs/root/version_history/current.rst

```diff
@@ -9,6 +9,7 @@ Incompatible Behavior Changes
 Minor Behavior Changes
 ----------------------
 *Changes that may cause incompatibilities for some users, but should not for most*
+* network.ext_authz filter append destination hostname for authorization service to review.
```

Comment:
Usually, we do something like: 

```
* ext_authz: added requested server name ...
```

Reply:
much simpler, still want to keep "network." prefix, since we have http.ext_authz as well.

Reply:
Yeah sorry for this nit. You can add that in the sentence after `:`, e.g. https://github.com/envoyproxy/envoy/blob/main/docs/root/version_history/v1.21.0.rst#bug-fixes see the ext_authz entry there.